src/inject/dynamic-theme/stylesheet-modifier.ts

```diff
@@ -30,8 +30,11 @@ export function createStyleSheetModifier() {
     const rulesModCache = new Map<string, ModifiableCSSRule>();
     const varTypeChangeCleaners = new Set<() => void>();
     let prevFilterKey: string = null;
-    let foundNonLoadedLinks = false;
+    // 'false' It means Dark Reader didn't find any non loaded link(s) on this stylesheet.
```

Comment:
This comment seems like not adding any details to the variable name and type. So you can remove it, or explain why these two booleans are important, or make variables' names more descriptive.